SignalServiceKit/src/Contacts/Contact.m

```diff
@@ -133,6 +134,23 @@ - (instancetype)initWithSystemContact:(CNContact *)contact
     return self;
 }
 
+- (instancetype)initWithCoder:(NSCoder *)coder
+{
+    self = [super initWithCoder:coder];
+    if (_imageData) {
+        _image = [UIImage imageWithData:_imageData];
```

Comment:
This is a way for `isEqual:` to work when determining if two Contact's are equivalent.